langchain/src/schema/index.ts

```diff
@@ -723,8 +724,29 @@ export abstract class BaseEntityStore extends Serializable {
  * Abstract class for a document store. All document stores should extend
  * this class.
  */
-export abstract class Docstore {
+export abstract class Docstore extends BaseStore<string, Document> {
+  lc_namespace = ["langchain", "schema"];
+
   abstract search(search: string): Promise<Document>;
 
   abstract add(texts: Record<string, Document>): Promise<void>;
+
+  async mget(keys: string[]): Promise<Document[]> {
+    return Promise.all(keys.map((key) => this.search(key)));
```

Comment:
Discussed offline, this probably won't work since `search` may be fuzzy search through content (like a vectorstore)
rather than get page by id